crates/control/src/led_status.rs

```diff
@@ -199,6 +202,16 @@ impl LedStatus {
                 } else {
                     None
                 };
+                let right_eye = match role {
+                    Role::Loser => Eye::loser_eye(),
+                    Role::Striker => Eye::striker_eye(),
+                    Role::StrikerSupporter => Eye::striker_support_eye(),
+                    Role::DefenderLeft | Role::DefenderRight => Eye::defender_eye(),
```

Comment:
Could we maybe split this up, and split the colours in the eye based on the defender side? Might be useful!